sdc/tests/tests_perf/test_perf_df.py

```diff
@@ -35,46 +34,10 @@
 from sdc.tests.tests_perf.test_perf_base import TestBase
 from sdc.tests.tests_perf.test_perf_utils import calc_compilation, get_times, perf_data_gen_fixed_len
 from sdc.tests.test_utils import test_global_input_data_float64
-from sdc.io.csv_ext import to_varname
+from .generator import *
```

Comment:
`from <module> import *` is bad code style. Please, list what you have imported from the module.